scripts/postprocess.py

```diff
@@ -352,14 +352,29 @@ def parse_jobs(data_dirs, distinguish_dir=False):
     if not isinstance(data_dirs, list):
         data_dirs = [data_dirs]
     data = []
-    for data_dir in data_dirs:
+    for data_dir_label in data_dirs:
+        dir_label = data_dir_label.split(":")
```

Comment:
I don't know what the most appropriate error checking/reporting thing is for python, but could we at least say `assert(1 <= len(dir_label) <= 2)` or whatever the python way of doing that is?